crowbar_framework/app/controllers/repositories_controller.rb

```diff
@@ -76,12 +78,14 @@ def activate
   # required parameters: platform, repo
   def deactivate
     return render_not_found if params[:platform].nil? || params[:repo].nil?
+    ret, message = ProvisionerService.new(logger).disable_repository(params[:platform], params[:repo])
```

Comment:
Lint/UselessAssignment: Useless assignment to variable - `message`. Use `_` or `_message` as a variable name to indicate that it won't be used. (https://github.com/bbatsov/ruby-style-guide#underscore-unused-vars)